### PR TITLE
Initial Commit Small Hammer

### PR DESCRIPTION
**Pull request by Meowkov**
Added Small Hammer

### CODE REVIEW

**Review by Meowkov — COMMENTED** [review]
Looking good.
Approved on my end.
Hammerssassins go!

**Review by Yeldur — DISMISSED** [review]
Approving on Meow's behalf

**Review by Erdertus — CHANGES_REQUESTED** [review]
+3 tier is too high. If it's a struggle to make it fit because presumably dropping weight is costing a lot at this point, maybe just drop the +0 to T9 or something.

**Comment by Yeldur** [comment]
> +3 tier is too high. If it's a struggle to make it fit because presumably dropping weight is costing a lot at this point, maybe just drop the +0 to T9 or something.

I disagree that +3 is too high, it's 10.2, barely over t10 and gains +1 damage which is _all_ it gains at +3 - This is a minor increase overall. Lowering to t9 makes no logical sense simply because +3 goes a marginal amount over the line.

**Comment by Yeldur** [comment]
Discussed tiers within IB team and made decision of allowing up to t10.2 without justification, as a result resubmitting review request

File: items.json [hunk 1]
```diff
@@ -40113,7 +40113,7 @@
     "weight": 3.1,
     "tier": 7.56281948,
     "rank": 3,
-    "requirement": 15,
+    "requirement": 18,
```
Comment (Muparadzi) [inline items.json:40116]: Remove these changes whilst we wait for #473 to be resolved.

**Review by Muparadzi — CHANGES_REQUESTED** [review]

**Review by Muparadzi — APPROVED** [review]

**Review by Erdertus — APPROVED** [review]

**Review by Yeldur — DISMISSED** [review]
After double checking things with Meow and testing myself, I'm happy - Approving

**Comment by Yeldur** [comment]
Rebase required following merge of https://github.com/namidaka/itembalancing/pull/476

**Review by Yeldur — APPROVED** [review]
rebase should have added the xbow changes as they are part of current master branch -> issues resolved -> approved

**Pull request merged**